bin/lightning-scan.js

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env node
+
+var program = require('commander'),
+    request = require('request');
+
+var outputSuccessMessage = require('../lightning/output').outputSuccessMessage;
+
+const LIGHTNING_WEBAPP_BASE_URL = 'http://osylvain.github.io/lightning/';
+const LIGHTNING_TASK_URL        = 'https://v8acs2yqh4.execute-api.us-east-1.amazonaws.com/prod/scan';
+
+function immediateScan(targetUrl) {
+  if (!targetUrl || typeof targetUrl != "string") console.log('a valid target URL needs to be specified');
+  else {
+    console.log('scanning ', targetUrl, ' ...');
+
+    var requestOptions = {
+      json: {url: targetUrl}
+    };
+
+    // TODO: wrap request in a promise and extract to a index.js module so lightning can be called by via node
+    request.post(LIGHTNING_TASK_URL, requestOptions, function (err, httpResponse, body) {
+      if (err) return console.error('Oups, somethig went wrong while scanning ...', err.message)
+      else if (body.message == 'Endpoint request timed out') { // API GATEWAY TIMES OUT AFTER 10 seconds
+        outputSuccessMessage('Perf scan not complete yet, your results will likely be available at ' + LIGHTNING_WEBAPP_BASE_URL + ' in less than a minute');
+      } else if (body.metrics) {
+        outputSuccessMessage('Perf scan complete, go check your results at ' + LIGHTNING_WEBAPP_BASE_URL);
+        for (metric in body.metrics) {
```

Comment:
Could add a verbose arg and display metrics accordingly. Or have them displayed by default and have a silent or quiet arg. Just that it seems annoying to always have a 200 lines output every time I scan. What do you think?